core/src/test/java/apoc/warmup/WarmupEnterpriseTest.java

```diff
@@ -0,0 +1,41 @@
+package apoc.warmup;
+
+import apoc.util.Neo4jContainerExtension;
+import apoc.util.TestContainerUtil;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.neo4j.configuration.GraphDatabaseInternalSettings;
+import org.neo4j.configuration.GraphDatabaseSettings;
+import org.neo4j.driver.Session;
+
+import java.util.List;
+
+import static apoc.util.TestContainerUtil.createEnterpriseDB;
+import static apoc.util.TestContainerUtil.testCall;
+
+public class WarmupEnterpriseTest {
+
+    private static Neo4jContainerExtension neo4jContainer;
+    private static Session session;
+
+    @BeforeClass
+    public static void beforeAll() {
+        neo4jContainer = createEnterpriseDB(List.of(TestContainerUtil.ApocPackage.CORE), true)
+                .withNeo4jConfig(GraphDatabaseInternalSettings.include_versions_under_development.name(), "true")
+                .withNeo4jConfig(GraphDatabaseSettings.db_format.name(), "freki");
+        neo4jContainer.start();
+        session = neo4jContainer.getSession();
+    }
+
+    @AfterClass
+    public static void afterAll() {
+        session.close();
+        neo4jContainer.close();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void testWarmupIsntAllowedWithOtherStorageEngines() {
+        testCall(session, "CALL apoc.warmup.run()", (r) -> {});
```

Comment:
Two nitpicks:
* We are missing a positive test
* Does this need to be an integration test (with a docker container)? Isn't there a way to test this with a unit test?

Reply:
I can add specific positive tests :)

The only way to test the negative case was to use an enterprise db, is there a way to get a test on that without docker? When I searched for setting up enterprise in other tests this was how they did it, but maybe I misunderstood? If so let me know.

Reply:
That sounds good :) I can make a card for finding that out